src/views/CreateView.vue

```diff
@@ -91,18 +97,18 @@
                           class="absolute right-2 h-8"
                           >
                           <img
-                            :src="exercise.img"
+                            :src="exercise.exercise.img"
```

Comment:
This can be avoided with object destruct

Reply:
**Corrected**

Not sure if I did it in the way you wanted me to though.